### PR TITLE
[BUGFIX] fix grammar + fix missing pixels on 'DROPLET' symbol

### PR DESCRIPTION
**Pull request by Clumxy**
## About The Pull Request

The pull request in particular replaces the word 'kinda' with the word 'kind of'. This change is intended to enhance grammar and formality. 

## Proof of Testing

image

## Changelog/Credits

Bug fixes I guess?

### CODE REVIEW

**Comment by Clumxy** [comment]
id also like to say that i edited a symbol with missing pixels.

thanks for https://github.com/ClanGenOfficial/clangen/issues/2886 for bringing this to my attention!
image

**Comment by Clumxy** [comment]
Bumping, since I havent gotten a review yet.

File: resources/dicts/patrols/forest/hunting/newleaf.json [hunk 1]
```diff
@@ -3483,7 +3483,7 @@
                 ]
             },
             {
-                "text": "It's a wonderful day, wandering through the sunshine kinda randomly around c_n's forest searching for the signs of bird nests and swarming up the trees for them. s_c is even able to show off a little, inching cautiously out onto the spindly branches of a willow to snag a hard-to-reach nest.",
+                "text": "It's a wonderful day, wandering through the sunshinekind of randomly around c_n's forest searching for the signs of bird nests and swarming up the trees for them. s_c is even able to show off a little, inching cautiously out onto the spindly branches of a willow to snag a hard-to-reach nest.",
```
Comment (tobes03) [inline resources/dicts/patrols/forest/hunting/newleaf.json]: I noticed you're missing a space here between "sunshine" and "kind"

**Comment by scribblecrumb** [comment]
> Bumping, since I havent gotten a review yet.

@Clumxy As I said on your last PR, please don't bump your PRs.  Senior devs will review when they have time, bumping does not make your PR "more visible" or encourage us to review it quicker.  I suggest having patience, senior devs are just as much volunteers as you are.

File: resources/dicts/patrols/forest/hunting/newleaf.json (hunk 1 above)
Comment (Clumxy) [reply resources/dicts/patrols/forest/hunting/newleaf.json]: Working on!

**Review by anjubatus — APPROVED** [review]

**Pull request merged**